pysb/pathfinder.py

```diff
@@ -77,7 +89,21 @@
             'posix': ('/usr/local/share/StochKit',),
             'nt': ('c:/Program Files/StochKit',)
         }
+    },
+    'nvcc': {
+        'name': 'NVIDIA CUDA compiler',
+        'executable': {
+            'posix': 'nvcc',
+            'nt': 'nvcc.exe'
+        },
+        'env_var': 'CUDAPATH',
+        'search_paths': {
+            'posix': ('/usr/local/cuda/bin', '/usr/bin', '/opt/cuda/bin'),
+            'nt': (
+            r'C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\bin',)
```

Comment:
Probably better to let the user use `CUDAPATH` than to hard code a location here. Particularly now since the user's `PATH` is searched.

Reply:
Should I remove the search paths then?